django-verdant/rca/migrations/0064_auto_20170124_0935.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.9 on 2017-01-24 09:35
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0029_unicode_slugfield_dj19'),
+        ('rca', '0063_auto_20170124_0918'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ProgrammePageKeyContent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('sort_order', models.IntegerField(blank=True, editable=False, null=True)),
+                ('link_text', models.CharField(blank=True, help_text=b'', max_length=255)),
+                ('link', models.ForeignKey(blank=True, help_text=b'', null=True, on_delete=django.db.models.deletion.CASCADE, related_name='+', to='rca.StandardPage')),
+            ],
+            options={
+                'ordering': ['sort_order'],
+                'abstract': False,
+            },
+        ),
+        migrations.AddField(
```

Comment:
This field was just deleted in https://github.com/torchbox/verdant-rca/pull/98/files#diff-1a0a34590e830aaede2bf11658679d04R19. Do we need to keep the previous content?